erpnext/buying/doctype/purchase_order/purchase_order.py

```diff
@@ -103,10 +103,7 @@ def get_last_purchase_rate(self):
 					d.price_list_rate = d.base_price_list_rate / conversion_rate
 					d.rate = d.base_rate / conversion_rate
 				else:
-					# if no last purchase found, reset all values to 0
-					for field in ("base_price_list_rate", "base_rate",
-						"price_list_rate", "rate", "discount_percentage"):
-							d.set(field, 0)
+					msgprint("Last purchase rate not found")
```

Comment:
add strings with _ so they become translatable like

`msgprint("Last purchase rate not found")`